spec/quotas/quotas.mdown

```diff
@@ -0,0 +1,147 @@
+# Introduction
+
+JMAP ([@!RFC8620] – JSON Meta Application Protocol) is a generic protocol for synchronising data, such as mail, calendars or contacts, between a client and a server. It is optimised for mobile and web environments, and aims to provide a consistent interface to different data types.
+
+This specification defines a data model for handling mail quotas over JMAP, allowing you to read and explain quota information.
+
+This specification does not address quota administration, which should be handled by other means.
+
+## Notational conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in BCP
+14 [@!RFC2119] [@!RFC8174] when, and only when, they appear in all
+capitals, as shown here.
+
+Type signatures, examples and property descriptions in this document follow the conventions established in section 1.1 of [@!RFC8620]. Data types defined in the core specification are also used in this document.
+
+Servers MUST support all properties specified for the new data types defined in this document.
+
+## Terminology
+
+### Quota
+
+A quota is a numeric upper limit that the server is enforcing. Quotas are applied to JMAP Mailbox objects.
+
+## Addition to the capabilities object
+
+The capabilities object is returned as part of the JMAP Session object; see [@!RFC8620], section 2.
+
+This document defines one additional capability URI.
+
+### urn::ietf::params::jmap::quota
+
+This represents support for the Quota data type and associated API methods. Servers supporting this specification MUST add this property to the capabilities object.
+
+# Quota definition
+
+The quota is an object that displays the limit set to a mailbox usage as well as the current usage in regard to that limit.
+
+## The Scope Data Type
+
+The **Scope** is a `String` from an enumeration defined list of values, handled by the server.
```

Comment:
does the scope of any use for "used" field? I mean, I never saw a quota for a domain, it's always something personal

Reply:
Technically from an IMAP RFC point of view, a "quotaRoot" could be imagined to match the domain, both for "limit" and "used" for instance.

Corner case, not common, but I see no reason to be restrictive here.